### ISSUE TITLE
上传一个41字节的文本文件，下载回来是256字节？

### ISSUE THREAD

**Issue by andy65007** [caozhiyuan/FastDFSNetCore#13]
调用库后，改变了原有文件内容，不太好啊

**Comment by caozhiyuan** [comment]
你可以手动调fastdfsclient 上传试下

**Comment by andy65007** [comment]
手动调？啥意思？

**Comment by andy65007** [comment]
下面代码有问题吗？
`List<IPEndPoint> pEndPoints = new List<IPEndPoint>()
{
	new IPEndPoint(IPAddress.Parse("192.168.80.5"), 22122)
};
ConnectionManager.Initialize(pEndPoints);

string fileName = "test.txt";
string path = Path.Combine("D:\\", fileName);

StorageNode node = FastDFSClient.GetStorageNodeAsync().GetAwaiter().GetResult();
string s = FastDFSClient.UploadFileAsync(node, new MemoryStream(File.ReadAllBytes(path)), "txt").GetAwaiter().GetResult();;

Console.WriteLine(s);


byte[] fileData = FastDFSClient.DownloadFileAsync(node, s).GetAwaiter().GetResult();
Console.WriteLine("length:" + fileData.Length);

File.WriteAllBytes("D:\\123.txt", fileData);

            `

**Comment by caozhiyuan** [comment]
please use 
```
Task DownloadFileAsync(StorageNode storageNode, string fileName,
            IDownloadCallback downloadCallback,
            long offset = 0,
            long length = 0)
```

i will fix return byte array bug

**Comment by caozhiyuan** [comment]
@andy65007  fixed in 1.2.4 , suggest use stream for big file.

**Comment by andy65007** [comment]
good!
